<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Goods</class>
 <widget class="QWidget" name="Goods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="l_pic">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>105</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>pic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_buy">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>购买</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_fav">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>收藏</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_price">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>price</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_agio">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>agio</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
